sdk/core/Azure.Core/api/Azure.Core.netstandard2.0.cs

```diff
@@ -569,3 +570,29 @@ public abstract partial class ObjectSerializer
         public virtual System.Threading.Tasks.ValueTask<System.BinaryData> SerializeAsync(object? value, System.Type? inputType = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
 }
+namespace Azure.Messaging
+{
+    [System.Text.Json.Serialization.JsonConverterAttribute(typeof(Azure.Messaging.CloudEventConverter))]
+    public partial class CloudEvent
+    {
+        public CloudEvent(string source, string type, object jsonSerializable, System.Type? dataSerializationType = null) { }
```

Comment:
how about a generic `CloudEvent<T>` class to support strong type data, while this class is a simple representation of CloudEvent data structure?

Reply:
The way we support strong data type is through the [BinaryData](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/core/System.Memory.Data#binary-data) type which is what the `Data` property is typed as.